actions/build-push-docker/README.md

```diff
@@ -0,0 +1,99 @@
+# build-push-docker
+
+## Example usage
+
+**NOTE**: _This composite workflow is intended to be used in conjunction with
+`../cicd-build-docker-manifest-push` which will create a Docker manifest (or
+index) of the images created within this composite workflow._
+
+### Set the following repo secrets
+
+**NOTE**: _Requires the [gh cli](https://cli.github.com/)._
```

Comment:
Might want to link here: [smartcontract-it.atlassian.net/wiki/spaces/RE/pages/906985607/GitHub+Repo+Configuration](https://smartcontract-it.atlassian.net/wiki/spaces/RE/pages/906985607/GitHub+Repo+Configuration)

Reply:
Avoiding due to this being a public repo.